go-controller/pkg/ovn/egressip.go

```diff
@@ -1600,6 +1600,14 @@ func (oc *Controller) setNodeEgressReachable(nodeName string, isReachable bool)
 
 func (oc *Controller) addEgressNode(nodeName string) error {
 	var errors []error
+	// Check if EgressIP node create failed and if does try adding it again
+	if node, ok := oc.addEgressIPNodeFailed.Load(nodeName); ok {
+		failedNode := node.(*kapi.Node)
+		if err := oc.addNodeForEgress(failedNode); err != nil {
```

Comment:
unrelated to your PR, Mohamed, but... can we rename  `addNodeForEgress`? It's terribly confusing since we have also `addEgressNode`. Can it be something like `initEgressNode`? Or anything else that doesn't include the string "add" :)   @kyrtapz